events/models.py

```diff
@@ -440,6 +442,11 @@ class SuperEventType:
     has_start_time = models.BooleanField(default=True)
     has_end_time = models.BooleanField(default=True)
 
+    # In accordance with this convention schema.org/typicalAgeRange
```

Comment:
It could have a validator to enforce the convention.
Is indexing the field necessary?